soft4pes/utils/sequence.py

```diff
@@ -46,9 +46,13 @@ def __call__(self, t):
             Interpolated output.
 
         """
-        interpolated_values = []
+
```

Comment:
Above ``times`` shouldn't be "sampled time" or "time instants"?

Reply:
Resolved in the next commit